File: C++/All Codes/Divisible7.cpp

#include<iostream>
using namespace std;
class Divisible
{
    int num;
    public: void check()
    {
        cout<<"Enter the number\n";
        cin>>num;
     if(num%7==0)
     {
         cout<<"Number is divisible by 7\n";
     }
     else{
        cout<<"Not divisible by 7\n";
     }
    }
};
int main()
{
    Divisible d1;
    d1.check();
}

File: JAVA/All Codes/Divisible_7.java
import java.util.Scanner;  
class Divisible_7  
{ 
public static void main(String args[])
{
int num; 
System.out.println("Enter the Number:");
Scanner s=new Scanner(System.in); 
num=s.nextInt();
if(num%7==0)
{ 
System.out.println("................................\n");
System.out.println("Number is Divisible by 7");
}
else
{
System.out.println("................................\n");
System.out.println("Number is not Divisible by 7");
} 
} 
}